src/fr/charly/javaTutorial/JavaLesson15.java

```diff
@@ -0,0 +1,57 @@
+package fr.charly.javaTutorial;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.StandardCopyOption;
+import java.util.Arrays;
+import java.util.List;
```

Comment:
There are to many unused imports. Please, do a Ctrl+maj+o to remove all of them.